src/components/business-lookup/BusinessLookup.vue

```diff
@@ -201,27 +200,30 @@ export default class BusinessLookup extends Vue {
   }
 
   /** Sets the business name. */
-  protected setBusinessName (val: string): void {
+  setBusinessName (val: string): void {
     const name = val?.trim()
     this.onSelectedBusiness({ ...this.businessLookup, name } as any)
   }
 
   /**
-   * Called when searchField property has changed.
+   * Called when Search Input has been updated.
    * This method is debounced to prevent excessive calls to the API.
    */
-  @Watch('searchField')
-  private onSearchField = debounce(async () => {
+  onSearchInput (searchInput: string) {
+    this.onSearchInputDebounced(this, searchInput)
```

Comment:
Note that I pass in `this` since the function inside debounce has a different "this" scope.

And below, I accept "this" and call it "that".